### PR TITLE
Changed leaderboard: menu, banners, animation, extra page

### PR DESCRIPTION
**Pull request by aidera**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -115,7 +116,16 @@ function App() {
           <Route path="/balance" exact element={<Balance />} />
           <Route path="/point-tracker" exact element={<Referral />} />
           <Route path="/task" exact element={<TaskPage />} />
-          <Route path="/leadboard" exact element={<Leaderboard />} />
+          <Route
+            path="/leadboard"
+            exact
+            element={<Navigate to="/leadboard/league" />}></Route>
```
Comment (fifa-master) [inline src/App.js:122]: correct?

Comment (aidera) [reply src/App.js:122]: yes, 2 pages now. And let it be redirect

File: src/pages/leaderboard/assets/Banner1/Banner1.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export default function Banner1() {
+  const timer = useWeeklyTimer();
+
+  const week = useMemo(() => {
+    return getWeekNumber(new Date(Date.UTC(2024, 6, 1)));
+  }, []);
+
+  return (
+    <div className={styles.banner}>
+      <div className={styles['banner-image']}>
```
Comment (fifa-master) [inline src/pages/leaderboard/assets/Banner1/Banner1.jsx:15]: dont use `-` in classname

Comment (aidera) [reply src/pages/leaderboard/assets/Banner1/Banner1.jsx:15]: Why not? :)

File: src/pages/leaderboard/assets/Table/Table.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    );
+
+    notAnimatedLeaders.forEach((el, index) => {
+      setTimeout(() => {
+        setAnimatedLeaderIds((prevIds) => new Set(prevIds).add(el.id));
+      }, index * 80);
+    });
+  }, [items]);
+
+  function anonymizeFullName(str) {
```
Comment (fifa-master) [inline src/pages/leaderboard/assets/Table/Table.jsx:21]: move to utils folder

File: src/pages/leaderboard/friends/index.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import { getLeaderboardEffect } from '../../../effects/leaderboardEffect';
+import Banner2 from '../assets/Banner2/Banner2';
+import Menu from '../../../components/Menu/Menu';
+import Header from '../assets/Header/Header';
+import Table from '../assets/Table/Table';
+import { ReactComponent as LoaderIcon } from '../../../assets/loader.svg';
+
+import style from '../style.module.css';
+
+export const LeaderboardFriends = () => {
```
Comment (fifa-master) [inline src/pages/leaderboard/friends/index.jsx:12]: combine into one
LeaderboardLeague and LeaderboardFriends

File: src/utils/useWeeklyTimer.jsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react';
+
+const useWeeklyTimer = () => {
+  // Calculate time left until next Monday Central Time
+  const calculateTimeLeft = () => {
```
Comment (fifa-master) [inline src/utils/useWeeklyTimer.jsx:5]: usecallback

**Review by fifa-master — APPROVED** [review]

**Pull request merged**